src/main/java/jp/co/future/uroborosql/exception/OptimisticLockException.java

```diff
@@ -15,8 +15,9 @@
  */
 public class OptimisticLockException extends UroborosqlRuntimeException {
 
-	public OptimisticLockException(SqlContext context) {
-		super(String.format("An error occurred due to optimistic locking.\nExecuted SQL [\n%s]\nparams:%s", context.getExecutableSql(), context.formatParams()));
+	public OptimisticLockException(final SqlContext context) {
+		super(String.format("An error occurred due to optimistic locking.\nExecuted SQL [%n%s]%nparams:%s",
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [FS: Format string should use %n rather than n (VA_FORMAT_STRING_USES_NEWLINE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#fs-format-string-should-use-n-rather-than-n-va-format-string-uses-newline).
This format string includes a newline character (\n). In format strings, it is generally  preferable to use %n, which will produce the platform-specific line separator.